src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -179,9 +183,29 @@ def _fill_ftp_publishing_url(webapp, resource_group_name, name, slot=None):
     return webapp
 
 
+def _format_linux_fx_version(custom_image_name):
+    fx_version = custom_image_name
+    fx_version_lower = fx_version.lower()
+    # handles case of only spaces
+    if custom_image_name.strip():
```

Comment:
If the custom-image name is a number of spaces, we want fx_version to become `' '`